README.adoc

```diff
@@ -285,6 +285,102 @@ spec:
 ...
 ----
 
+== Configure Health
+
+=== Kiali defaults
+
+By default Kiali health is:
```

Comment:
This is not clear.
It's assuming what is health in all context.
Health happens in several places: Graph, Namespace, Application, Workload and Service.
And for each component there are several items that are used to compute them (istio metrics and also controller data current pods, requested, etc).

This need to be documented properly in a way that the user understand what is referring to.

Reply:
This impact in  Graph, Namespace, Application, Workload and Service